user-service/README.MD

```diff
@@ -0,0 +1,19 @@
+# PeerPrep Backend for User Service - Developer Readme 
+
+## Quick Start
+
+- This backend service uses <b>npm</b> as the package manager
+- Why not yarn? Because i don't like.
+
+### How to run backend server
+
+1. Add the `USER_SERVICE_MONGODB_URL` (which max hasn't sent in tele chat) into the `.env` file
+2. Run `npm run dev` (make sure you are in the `backend/user-service/` directory)
```

Comment:
one more thing, pls add an `npm i` instruction here to install dependencies:

```diff
- 1. Add the `USER_SERVICE_MONGODB_URL` (which max hasn't sent in tele chat) into the `.env` file
- 2. Run `npm run dev` (make sure you are in the `backend/user-service/` directory)
+ 1. Run `npm i` in the `backend/user-service` directory to install dependencies
+ 2. Add the `USER_SERVICE_MONGODB_URL` (which max hasn't sent in tele chat) into the `.env` file
+ 3. Run `npm run dev` (make sure you are in the `backend/user-service/` directory)
```

Reply:
ok fixed.